lib/models/operation.js

```diff
@@ -30,10 +30,10 @@ class Operation extends OperationTraitable {
    * @returns {Message}
    */
   message(index) {
-    if (!this._json.message) return null;
+    if (typeof index !== 'number' || !this._json.message) return null;
     if (!this._json.message.oneOf) return new Message(this._json.message);
     if (index > this._json.message.oneOf.length - 1) return null;
-    return new Message(this._json.message.oneOf[index]);
+    return new Message(this._json.message.oneOf[+index]);
```

Comment:
how is this change related to security warnings?

Reply:
`eslint-plugin-security` warns that index is custom variable passed by argument, and it could be string etc. `+` before variable, casts it to number: `"1" => 1`. If you pass something like `"FOO"`, then it will be `NaN`, so it removes malicious code written as string in variable.